pyproject.toml

```diff
@@ -3,16 +3,16 @@ requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "ifk-scb-compilations"
+name = "ifk_scb_compilations"
```

Comment:
I think the convention is to use dashes in project names (here) and underscores in package/module names. See also copier file.

Reply:
updated.